### PR TITLE
sectioning out backup and restore creation

### PR DESCRIPTION
**Pull request by hallaroo**
https://deploy-preview-728--vitess.netlify.app/docs/user-guides/operating-vitess/backup-and-restore/creating-a-backup/

### CODE REVIEW

File: content/en/docs/user-guides/operating-vitess/backup-and-restore/creating-a-backup.md [hunk 1]
```diff
@@ -130,6 +116,41 @@ Bootstrapping a new tablet is almost identical to restoring an existing tablet.
...
+## Common Errors and Resolutions
+
+### No xtrabackup User passed to vttablet
+
+```
+E0310 08:15:45.336083  197442 main.go:72] remote error: rpc error: code = Unknown desc = TabletManager.Backup on zone1-0000000102 error: xtrabackupUser must be specified.: xtrabackupUser must be specified
+```
+Fix: Set vttablet flag [-xtrabackup_user](../#basic-vttablet-configuration)
+
+### xtrabackup binary not found in path
```
Comment (gedgar) [inline content/en/docs/user-guides/operating-vitess/backup-and-restore/creating-a-backup.md]: To make this a bit more clear, we can change path to $PATH

**Review by gedgar — COMMENTED** [review]
LGTM (outside of one small nit)

**Review by Anders-PlanetScale — APPROVED** [review]
Approved

**Review by askdba — COMMENTED** [review]
LGTM

**Pull request merged**